container-images/aws-utils/index.ts

```diff
@@ -0,0 +1,26 @@
+import KMS from 'aws-sdk/clients/kms'
```

Comment:
This looks like an exact copy of what's in `@aws-lambda/aws-utils` which raises some questions. 1) Do we even need it? 2) Is there a better way to handle secrets/encryption in a docker image than using KMS? 3) Can this module just be imported from somewhere common?

Reply:
I can get rid of this - kubernetes has its own secret storage which I've updated the helm chart to use